tmt/base.py

```diff
@@ -319,6 +318,17 @@ def lint(self):
             echo(verdict(2, 'summary is very useful for quick inspection'))
         elif len(self.summary) > 50:
             echo(verdict(2, 'summary should not exceed 50 characters'))
+
+        # Convert test case relevancy into adjust rules if --fix enabled
```

Comment:
I'm missing Error if relevancy attribute is present and --fix is not used.

Reply:
As in lint error, eg
```
/test/a
pass test script must be defined
pass directory path must be defined
error uses obsolete relevancy attribute
```

Reply:
Good point! Fixed.